src/screens/UserPortal/Posts/Posts.module.css

```diff
@@ -6,14 +6,10 @@
   color: white;
 }
 
-.colorLight {
-  background-color: #f5f5f5;
-}
-
 .mainContainer {
   width: 50%;
   flex-grow: 3;
-  padding: 1rem;
+  padding: 20px;
```

Comment:
Why isn't this `rem`?  Will this be responsive

Reply:
I'm sorry for that, please check the latest changes.